docs/src/examples/components/Dropdown/Types/DropdownExampleClearable.shorthand.tsx

```diff
@@ -0,0 +1,20 @@
+import { Dropdown } from '@stardust-ui/react'
+import * as React from 'react'
+
+const inputItems = [
+  'Bruce Wayne',
+  'Natasha Romanoff',
+  'Steven Strange',
+  'Alfred Pennyworth',
+  `Scarlett O'Hara`,
+  'Imperator Furiosa',
+  'Bruce Banner',
+  'Peter Parker',
+  'Selina Kyle',
+]
+
+const DropdownClearableExample = () => (
+  <Dropdown clearable items={inputItems} placeholder="Select your hero" />
```

Comment:
Maybe we should add additional example for showing customization of the `clearableIndicator`?

Reply:
If we will have request for it, we can introduce it. For now, I am not sure that we need to add example for an every slot, it can make docs unusable ⛸